pkg/trace/pb/decoder_bytes.go

```diff
@@ -0,0 +1,169 @@
+package pb
```

Comment:
Is this code generated? IIRC you were mentioning that yes. If so, why doesn't it have the right headers? Either way, copyright might be nice.

Reply:
As discussed offline this is not generated.